lib/Models/Definition/addModelStrataView.ts

```diff
@@ -19,6 +19,8 @@ export default function addModelStrataView<
 ): ModelPropertiesFromTraits<InstanceType<T>>;
 export default function addModelStrataView<
   T extends TraitsConstructor<ModelTraits>
+  /* TODO: Fix this overload type */
+  /* eslint-disable-next-line @typescript-eslint/ban-types */
 >(model: Function, Traits: T): ModelPropertiesFromTraits<InstanceType<T>>;
```

Comment:
I couldn't easily find an appropriate type for `model` that would satisfy tsc since it seems to get treated as both a function and a class constructor